build-tools/attributions-generator.sh

```diff
@@ -7,7 +7,7 @@
 set -e
 set -x
 
-ATTR_GEN_IMG=f5devcentral/attributions-generator:latest
+ATTR_GEN_IMG=f5networksdevel/attributions-generator:latest
```

Comment:
@ssurenr , @veeramalla-f5 - The f5networksdevel repo should not be used for any public dependencies. You should upload the latest attributions-generator image from there into f5devcentral.

Reply:
We kept it here due to the access restrictions. We will move this back to f5devcentral shortly.